lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -435,7 +493,18 @@ def interleave(self, source_obj_ref, source_obj_name, fwd_shock_filename,
                                                            rev_shock_node, rev_shock_filename])
                         error_message += 'Forward Path {}, Reverse Path {}.'
                         error_message_bindings.extend([fwdpath, revpath])
-                        raise ValueError(error_message.format(*error_message_bindings))
+                        
+                        if reads_source == 'web':
+                            error_message += 'Forward File URL {}, Reverse File URL {}.'
+                            error_message_bindings.extend([fwdsource, revsource])
+
+                        if reads_source == 'staging':
```

Comment:
yea. I am aware of this. But I need to mock _get_staging_file_path return. And for this function, _get_staging_file_path needs to return 2 different path. I cannot test it properly.